frontend/src/components/input.tsx

```diff
@@ -29,27 +31,34 @@ export const Input = ({ sendMessage }: InputProps) => {
     [sendMessage, userInput],
   );
 
+  const loadPrompt = useCallback((suggestion: string) => {
```

Comment:
Unnecessary function declaration. ` setUserInput` could just be used directly in place of loadPrompt.